### PR TITLE
Cleanup/code and helptext details

### PR DESCRIPTION
**Pull request by schreibe**
Hi Artur (and Allin), perhaps you can run your tests, I don't know if it's possible on Windows. (Although in theory perhaps using the Windows subsystem for Linux, but anyway...)
Nothing substantial has changed with respect to the released 0.2 version - the sample script here had some weird stuff at the top which was really out of sync with the released version, I corrected that while I was at it.
I removed "const int" and "const bool" in light of our recent discussions.
thanks
sven

### CODE REVIEW

File: src/parallel_specs.inp [hunk 1]
```diff
@@ -1,25 +1,34 @@
...
-                                      const bool debugging,
+function void display_workers_output (int n_jobs,
+                                      bool debugging,
                                       scalar *error)
     /* Print results of workers. */
 
-    loop i=1..n_jobs -q
-        printf "Info: Output for argument set %d:\n", i
+    loop i = 1..n_jobs -q
+        printf "PS-Info: Output for argument set %d:\n", i
```
Comment (atecon) [inline src/parallel_specs.inp:7]: Hi Sven, why did you change "Info" to "PS-Info"?
I really recommend to follow standards here and that is usually the "Info" keyword for logging purpose. The reason is simple: filter large log-files by some standard keyword such as "Info", "Error" etc.

Comment (atecon) [reply src/parallel_specs.inp:7]: Hi Sven, thanks for the changes. As you see I have only a single comment.

One word on running tests: You can always run the ./tests/run_tests.inp file as every gretl script. You just need to set the working directory on your machine.

The tests run ok here.

**Comment by schreibe** [comment]
Am 10.03.2021 um 18:00 schrieb Artur Tarassow:
> *@atecon* commented on this pull request.
>
> ------------------------------------------------------------------------
>
> In src/parallel_specs.inp
> <https://github.com/atecon/parallel_specs/pull/3#discussion_r591698661>:
>
>>                                        scalar *error)
>       /* Print results of workers. */
>
> -    loop i=1..n_jobs -q
> -        printf "Info: Output for argument set %d:\n", i
> +    loop i = 1..n_jobs -q
> +        printf "PS-Info: Output for argument set %d:\n", i
>
> Hi Sven, why did you change "Info" to "PS-Info"?
> I really recommend to follow standards here and that is usually the
> "Info" keyword for logging purpose. The reason is simple: filter large
> log-files by some standard keyword such as "Info", "Error" etc.

Hi Artur, I'm putting Allin on cc directly, because I don't think he's
on github (or if he is, he is not registered for this repo).

I inserted "PS-" because I found the output confusing, it's not clear if
you're not the author who's emitting which messages. I also thought
about being more explicit than "PS",  but I believe that would overdo it.

If you prefer to have "Info" at the beginning of the line, we could swap
the parts to "Info-PS" or something.

thanks
sven

**Comment by atecon** [comment]
> Am 10.03.2021 um 18:00 schrieb Artur Tarassow:
> ***@***.**** commented on this pull request. ------------------------------------------------------------------------ In src/parallel_specs.inp <[#3 (comment)](https://github.com/atecon/parallel_specs/pull/3#discussion_r591698661)>: > scalar *error) /* Print results of workers. */ - loop i=1..n_jobs -q - printf "Info: Output for argument set %d:\n", i + loop i = 1..n_jobs -q + printf "PS-Info: Output for argument set %d:\n", i Hi Sven, why did you change "Info" to "PS-Info"? I really recommend to follow standards here and that is usually the "Info" keyword for logging purpose. The reason is simple: filter large log-files by some standard keyword such as "Info", "Error" etc.
> Hi Artur, I'm putting Allin on cc directly, because I don't think he's on github (or if he is, he is not registered for this repo). I inserted "PS-" because I found the output confusing, it's not clear if you're not the author who's emitting which messages. I also thought about being more explicit than "PS", but I believe that would overdo it. If you prefer to have "Info" at the beginning of the line, we could swap the parts to "Info-PS" or something. 

Thanks for your explanation, Sven. Yes, maybe "Info-PS" is better as one can still search within an eventual log-file for the pattern "Line starts with 'Info'".

I think Allin is not on github. The "gretl-project" user is actually currently me who also created the repo: https://github.com/gretl-project/

Best,
Artur

**Comment by schreibe** [comment]
Am 10.03.2021 um 19:53 schrieb Artur Tarassow:

> Thanks for your explanation, Sven. Yes, maybe "Info-PS" is better as one
> can still search within an eventual log-file for the pattern "Line
> starts with 'Info'".

OK, pushed. Thanks also for the hint about the tests.
-s

**Review by atecon — APPROVED** [review]
